test/test_helper.rb

```diff
@@ -2,6 +2,17 @@
 # See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
 # frozen_string_literal: true
 
+if RUBY_VERSION >= '2.4.0'
+  require 'warning'
+  Gem.path.each do |path|
+    Warning.ignore(//, path)
+  end
+
+  # this is to ignore warnings that happen on the CI only
+  # the site_ruby part of the path needs to be removed if it exists otherwise the CI keeps doing the warnings
+  Warning.ignore(//, Gem::RUBYGEMS_DIR.gsub(/site_ruby\//, ''))
+end
```

Comment:
You might be able to put this into a file similar to test/simplecov_test_helper.rb and require that file here and in `multiverse/suite.rb` to reduce duplication.

Reply:
good idea! I added that in 062b1c77308e6d16bc8c6353fefe4a2520ca4276